### DISCUSSION TITLE
How to install GP to the SIM card?

### DISCUSSION THREAD

**Discussion by achechet** [martinpaljak/GlobalPlatformPro#325] [General]
All discussion is about how to use but my question is how to install GP to the sim card?
Or how to implement GP on my SIM card?
I have demo source for my chip and I can initialize it

**Comment by martinpaljak** [comment]
If you have a chip with full source and your idea is to implement the GP specifications, I would start by getting the relevant GP (and ETSI, in case of SIM) specifications. You'll probably get more help from the vendor of the chip and the source code for the chip.
